src/DMXSerial2.cpp

```diff
@@ -805,6 +814,78 @@ void DMXSerialClass2::_processRDMMessage(byte CmdClass, uint16_t Parameter, bool
 
 // ADD: PARAMETER_DESCRIPTION
 
+    } else if (Parameter == SWAPINT(E120_SENSOR_DEFINITION)) { // 0x0200
+      if (CmdClass == E120_GET_COMMAND) {
+        if (_rdm.packet.DataLength != 1) {
+          // Unexpected data
+          nackReason = E120_NR_FORMAT_ERROR;
+        } else if (_rdm.packet.SubDev != 0) {
+          // No sub-devices supported
+          nackReason = E120_NR_SUB_DEVICE_OUT_OF_RANGE;
+        } else {
+          uint8_t sensorNr = _rdm.packet.Data[0];
+          if (sensorNr >= _initData->sensorsLength) {
```

Comment:
That's sort of my point, you can only have 254 sensors, but you could technically pass 255 into the system. It probably really wants to be at init time, but you ought to ignore a sensor 255, as that's reserved for "all sensors".